lib/core/remove-file.ts

```diff
@@ -0,0 +1,70 @@
+import fs from "fs";
+import path from "path";
+import glob from "glob";
+
+import { alerts } from "./alerts";
+import {
+  getScssPathFromTypeDefinition,
+  getTypeDefinitionPath
+} from "../typescript";
+
+/**
+ * Given a single file remove the generated types if they exist
+ *
+ * @param file the SCSS file to generate types for
+ *
+ */
+
+export const removeFile = (file: string): void => {
+  if (fs.existsSync(file)) {
+    return;
+  }
+  const path = getTypeDefinitionPath(file);
+  try {
+    if (fs.existsSync(path)) {
+      fs.unlink(path, err => {
```

Comment:
Since we are not awaiting for this work to happen, we should ensure all of this work is done sync (instead of async via callbacks). This will likely lead to issues, but will be masked since this is done before the type generation this will likely flakeunexpectedly.

Can we possibly use `unlinkSync` and ensure this is all sync?